sfja/src/components/Upload/PreviewBlankForm.js

```diff
@@ -23,12 +23,15 @@ class PreviewBlankForm extends React.Component {
       const id = this.props.parentData.id;
       const name = this.state.value;
 
-      fetch('http://localhost:5000/updateFormName/' + id +'/' + name, {
+      const body = {
+        form_id: id,
+        form_name: name,
+      };
+      fetch('http://127.0.0.1:5000/updateFormName', {
```

Comment:
@trish234 FYI just changing some of your code in this PR as well
We mostly pass parameters in via a JSON object so I just migrated a few of your calls over to JSON style